src/acceptance/README.md

```diff
@@ -50,7 +50,7 @@ The full set of config parameters is explained below:
 
 * `service_name` (required): The name of the registered auto-scaler service, use `cf marketplace` to determine the name.
 * `service_plan` (required): The plan name of the registered auto-scaler service, use `cf marketplace` to determine the plan.
-* `report_interval` (required): How frequently metrics are collected. This value must match the value configured in your deployment.
+* `aggregate_interval` (required): How frequently metrics are aggregated. This value must match the value configured in your deployment.
```

Comment:
@boyang9527  Just trying to understand how do we check StatWindow/BreachDuration >= aggregate_interval (and should not be > 60 S ) in APIServer validation. APIServer validates based on the policy and aggregate_interval is not part of the policy.